helm/fmtok8s-conference-chart/templates/gateway-dashboard.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  name: gateway
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name }}
+spec:
+  url: https://raw.githubusercontent.com/salaboy/fmtok8s-conference-chart/main/platform/grafana/dashboards/gateway.json
```

Comment:
I was trying to use the json schema on this file... but I got some parsing errors.

Reply:
If you want to try this is the schema:

```yaml
apiVersion: integreatly.org/v1alpha1
kind: GrafanaDashboard
metadata:
  name: gateway
  labels:
    app.kubernetes.io/instance: {{ .Release.Name }}
spec:
  json: >
    {}
```